packages/@lightningjs/ui-components/src/components/CardRadio/CardRadio.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import CardTitle from '../Card/CardTitle';
+import TextBox from '../TextBox';
+import * as styles from './CardRadio.styles.js';
+import Radio from '../Radio';
+
+export default class CardRadio extends CardTitle {
+  static get __componentName() {
+    return 'CardRadio';
+  }
+
+  static get __themeStyle() {
+    return styles;
+  }
+
+  static get properties() {
+    return [...super.properties, 'subtitle'];
+  }
+
+  static get tags() {
+    return [...super.tags, 'Subtitle', 'Radio'];
+  }
+
+  static _template() {
+    return {
+      ...super._template(),
+      Subtitle: {
+        type: TextBox,
+        signals: {
+          textBoxChanged: '_update'
+        }
+      }
+    };
+  }
+
+  _update() {
+    super._update();
+    this._updateSubtitle();
+    this._updateRadio();
+    this._updateDescriptionPosition();
+    this._updateSubtitlePosition();
+  }
+
+  _updateTitle() {
+    super._updateTitle();
+    this._Title.patch({
+      signals: {
+        textBoxChanged: '_update'
+      }
+    });
+  }
+
+  _updateSubtitle() {
+    this._Subtitle.patch({
+      content: this.subtitle,
+      style: {
+        textStyle: {
+          ...this.style.subtitleTextStyle,
+          wordWrapWidth: this._calculateTextWidth()
+        }
+      }
+    });
+  }
+
+  _updateRadio() {
+    const { iconWidth, iconHeight } = this.style;
+    const radioPatch = {
+      w: iconWidth,
+      h: iconHeight,
+      x: this.w - iconWidth - this.style.paddingHorizontal,
+      y:
+        (this._Title.style.textStyle.lineHeight - iconHeight) / 2 +
+        this.style.paddingVertical +
+        this.style.paddingVertical / 4,
+      checked: true
+    };
+    if (!this._Radio) {
+      radioPatch.type = Radio;
+    }
+    this.patch({ Radio: radioPatch, alpha: this.style.alpha });
```

Comment:
I think this alpha needs to be moved to the Radio's style object since the Radio component has this method
```js
  _updateOpacity() {
    this.applySmooth(this, { alpha: this.style.alpha });
  }
```